src/tic_tac_toe_pair/api.clj

```diff
@@ -0,0 +1,9 @@
+(ns tic-tac-toe-pair.api
+  (:require [ring.middleware.defaults :refer [wrap-defaults api-defaults]]      
+            [tic-tac-toe-pair.routes :refer [api-routes]]
+            [tic-tac-toe-pair.middleware :refer [wrap-500-catchall]]))
+
+(def app 
+  (-> api-routes
+      (wrap-defaults api-defaults )
+      wrap-500-catchall))
```

Comment:
Editors usually have plugins that will automatically trim excess whitespace and add a newline to the bottom of every file you save. You might want to consider that for your editor of choice.

Reply:
Thank you Cyrus. I will get these styling rules setup on VSCode.